index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="nav">
+        <a href="">video</a>
+        <a href="">sidebar</a>
+    </nav>
```

Comment:
Good use of the semantic tag. However, I believe can be done by wrapping the navigation elements in their appropriate tags.

Reply:
- [ ] Perfect 💯